components/gantry/oneaxis/oneaxis.go

```diff
@@ -449,25 +210,8 @@ func (g *oneAxis) IsMoving(ctx context.Context) (bool, error) {
 
 // ModelFrame returns the frame model of the Gantry.
 func (g *oneAxis) ModelFrame() referenceframe.Model {
-	if g.model == nil {
-		var errs error
-		m := referenceframe.NewSimpleModel("")
-
-		f, err := referenceframe.NewStaticFrame(g.name, spatial.NewZeroPose())
-		errs = multierr.Combine(errs, err)
-		m.OrdTransforms = append(m.OrdTransforms, f)
-
-		f, err = referenceframe.NewTranslationalFrame(g.name, g.axis, referenceframe.Limit{Min: 0, Max: g.lengthMm})
-		errs = multierr.Combine(errs, err)
-
-		if errs != nil {
-			g.logger.Error(errs)
-			return nil
-		}
-
-		m.OrdTransforms = append(m.OrdTransforms, f)
-		g.model = m
-	}
+	g.mu.Lock()
+	defer g.mu.Unlock()
```

Comment:
Again, I don't see how this mutex helps anything: we're immediately returning, so there's no chance of some other goroutine trying to do something in between two of the steps in this function. (The mutex certainly doesn't hurt: feel free to keep it this way if it gives peace of mind.)